examples/2D_acoustic_pulse/case.py

```diff
@@ -0,0 +1,115 @@
+import math
+import json
+
+# Numerical setup
+Nx = 99
+Ny = 99
+dx = 8./(1.*(Nx+1))
+
+alf_st = 0.4
+
+p_inf = 101325
+rho_inf = 1
+gam = 1.4
+
+c = math.sqrt(gam*(p_inf) / rho_inf)
+cfl = 0.3
+mydt = cfl * dx / c 
+Tfinal = 80*1 / c
+Nt = int(Tfinal/mydt)
+
+# Configuring case dictionary
+print(json.dumps({
+    # Logistics ================================================================
```

Comment:
It might be worthwhile to remove the "distracting characters," as mentioned in #609, to save work for whoever fixes it down the road and avoid creating more problems.

Reply:
I think these are fine since it will likely be a single find/replace that removes all of these distracting characters from all the case files (and elsewhere.